docs/integrations/sources/qualtrics.md

```diff
@@ -0,0 +1,64 @@
+# Qualtrics Source Connector for Airbyte
+
+This page contains the setup guide and reference information for the Qualtrics source connector.
+
+## Prerequisites
+
+Before setting up the Qualtrics connector, ensure you have the following:
+
+- A Qualtrics account
+- An API key for your Qualtrics account
+
+## Supported Sync Modes
+
+The Qualtrics source connector supports the following sync modes:
+
+| Feature | Supported? |
+| :--- | :--- |
+| Full Refresh Sync | Yes |
+| Incremental - Append Sync | Yes (only for responses) |
+
+## Supported Streams
+
+This connector outputs the following streams:
+
+- **Groups:** Retrieves information about the groups in your Qualtrics account.
+- **Surveys:** Retrieves information about the surveys in your Qualtrics account.
+- **Survey Questions:** Retrieves the questions for each survey.
+- **Survey Responses:** Retrieves the responses for each survey.
+
+### Stream Details:
+
+1. **Groups:**
+   - **Endpoint:** `/groups`
+   - **Primary Key:** `id`
+
+2. **Surveys:**
+   - **Endpoint:** `/surveys`
+   - **Primary Key:** `id`
+
+3. **Surveys Questions:**
+   - **Endpoint:** `/survey-definitions/{{ stream_partition.id }}/questions`
+   - **Primary Key:** `id`
+   - **Partitioned by:** `survey_id`
+
+4. **Surveys Responses:**
+   - **Endpoint:** `/surveys/{{ stream_partition.id }}/responses`
+   - **Primary Key:** `responseId`
+   - **Partitioned by:** `survey_id`
+   - **Incremental Sync:** Supported using the `_lastModifiedDate` cursor field.
+
+## Changelog
+
+| Version | Date       | Pull Request                                      | Subject                                    |
+|---------|------------|--------------------------------------------------|--------------------------------------------|
+| 0.1.0  | 2024-06-05 | [38751](https://github.com/airbytehq/airbyte/pull/38751) | Initial release of Qualtrics connector    |
```

Comment:
Please update to match latest version of the changelog + change this to the bottom of the page.
```
<details>
  <summary>Expand to review</summary>

| Version | Date       | Pull Request                                             | Subject                                                                 |
|:--------|:-----------| :------------------------------------------------------- |:------------------------------------------------------------------------|
| 0.3.2   | 2024-05-14 | [38144](https://github.com/airbytehq/airbyte/pull/38144) | Make connector compatible with Builder                                  |
| 0.3.1   | 2023-06-05 | [27002](https://github.com/airbytehq/airbyte/pull/27002) | Flag spec `api_key` field as `airbyte-secret`                           |
| 0.3.0   | 2023-05-30 | [22642](https://github.com/airbytehq/airbyte/pull/22642) | Add `idea_comments`, `idea_endorsements`, and `idea_categories` streams |
| 0.2.0   | 2023-05-26 | [26666](https://github.com/airbytehq/airbyte/pull/26666) | Fix integration test and schemas                                        |
| 0.1.0   | 2022-11-02 | [18883](https://github.com/airbytehq/airbyte/pull/18893) | 🎉 New Source: Aha                                                      |

</details>
```